modules/structs/user.go

```diff
@@ -33,6 +33,10 @@ type User struct {
 	Created time.Time `json:"created,omitempty"`
 	// Is user restricted
 	Restricted bool `json:"restricted"`
+	// Is user active
+	IsActive *bool `json:"active,omitempty"`
```

Comment:
Why ref?

Reply:
Yes, just use `bool` if it will always have a value.

Reply:
Sorry! I take my words back. For non-admin users, hide the field is better. A nil bool means `Don't know if it's active`. @6543

Reply:
In this case all of these fields should be omitted:

https://github.com/go-gitea/gitea/blob/8f0539c235d47954d54dcf25c08297ddf7260970/modules/convert/user.go#L56-L61

`IsAdmin` would be a pointer then and `IsAdmin` and `Language` would get the `omitempty` flag. `LastLogin` already has the flag.

Reply:
well if it is always false it does not leak any info ... onto normal users